nym-wallet/src/pages/bonding/node-settings/settings-pages/general-settings/InfoSettings.tsx

```diff
@@ -126,7 +137,7 @@ export const InfoSettings = ({ bondedNode }: { bondedNode: TBondedMixnode | TBon
             </Grid>
           </Grid>
         </Grid>
-        <Divider flexItem />
+        <Divider sx={{ width: '100%' }} />
```

Comment:
can you use `variant=fullWidth` instead of setting the width this way?

Reply:
It does not work for some reason